src/styles/main.scss

```diff
@@ -0,0 +1,75 @@
+$width-day: 100px;
+$gap: 1px;
+$start-day: (
+  mon 0,
+  tue ($width-day + $gap),
+  wed (($width-day * 2) + ($gap * 2)),
```

Comment:
This calculation logic can be entered inside the loop, it will be better that way